modules/ossm-rn-fixed-issues.adoc

```diff
@@ -15,7 +15,22 @@ Provide the following info for each issue if possible:
 ////
 
 The following issues have been resolved in the current release:
-//current release is 2.5.1/2.4.7/2.3.11 scheduled for April 18, 2024
+
+//current release is 2.5.2/2.4.8/2.3.12 scheduled for May 22, 2024
+
+* https://issues.redhat.com/browse/OSSM-6331[OSSM-6331] Previously, invalid `LogLevel` values were accepted in `.smcp.general.logging.componentLevels` and the `ServiceMeshControlPlane` was created. Now, an error message is shown if an invalid value is used, and the control plane is not created.
+
+* https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] Previously, the **Project** filter of the **Istio Config** list page did not work correctly. All `istio config` items were displayed from all namespaces even if you select a specific project from the dropdown. Now, only the `istio config` items belonging to the selected project in the filter dropdown are displayed.
```

Comment:
```suggestion
* https://issues.redhat.com/browse/OSSM-6290[OSSM-6290] Previously, the **Project** filter of the **Istio Config** list page did not work correctly. All `istio config` items were displayed from all namespaces even if you selected a specific project from the dropdown. Now, only the `istio config` items that belong to the selected project in the filter dropdown are displayed.
```

Reply:
"Dropdown" usage note from ISG that may or may not impact this:

> ⚠️ **dropdown** _noun_ Use only in the context of Carbon Design or if its use is already established for a product. Although dropdown is prevalent in the industry, it might cause problems for translators and readers whose first language is not English.